test/metabase/test.clj

```diff
@@ -314,3 +315,11 @@
 (alter-meta! #'with-temp update :doc str "\n\n  Note: by default, this will execute its body inside a transaction, making
   it thread safe. If it is wrapped in a call to [[metabase.test/test-helpers-set-global-values!]], it will affect the
   global state of the application database.")
+
+;; Cursive does not understand p/import-macro, so we just proxy this manually
```

Comment:
Cursive implements its own internal macro support, which isn't extensible. It doesn't understand `import-macro`, so the usages show up as undefined references. Since there's only a single macro being exported, I felt that it's fine to just inline.

Some influential Clojurians think that inlining is preferable, and I'm weakly inclined that way too: https://clojureverse.org/t/a-less-controversial-alternative-to-potemkin/10563/3. Potemkin itself alludes to the controversy with its own tagline: "Some ideas which are almost good".

I don't mind that much either way, I'm slowly transitioning to Emacs for most of my coding 🙂

Reply:
Hmm well to me `import-vars` solves a real problem: wrapper functions / macros like this duplicate docstrings and names which require more effort to keep in sync.

Given you're switching to cursive anyway, and this problem exists already for all other macros with `import-vars`, I'd say let's at least be consistent and export it with `import-vars`?